.github/workflows/pr_checks.yml

```
@@ -0,0 +1,437 @@
+    if: ${{ always() }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set job status variables
+        id: set-status
+        run: |
+          IS_BOT="${{ needs.initial-author-check.outputs.is_bot }}"
+          echo "is_bot=$IS_BOT" >> $GITHUB_OUTPUT
+
+      # Show status report of all jobs
+      - name: Generate status report
+        run: |
```

actionlint (1.7.4)

property "changelog-check-user" is not defined in object type {changelog-update-dependabot: {outputs: {changelog_updated: string; dep_type: string; from_version: string; package_name: string; to_version: string}; result: string}; initial-author-check: {outputs: {is_bot: string; is_dependabot: string}; result: string}; metadata-milestone-check: {outputs: {}; result: string}; quality-lint-format-bot: {outputs: {}; result: string}; quality-lint-format-user: {outputs: {}; result: string}}

(expression)